tests/metrics/regression/test_ssim.py

```diff
@@ -50,7 +51,7 @@ def _sk_metric(preds, target, data_range, multichannel):
     [(i.preds, i.target, i.multichannel) for i in _inputs],
 )
 class TestSSIM(MetricTester):
-    atol = 1e-3  # TODO: ideally tests should pass with lower tolerance
+    atol = 6e-5  # TODO: ideally tests should pass with lower tolerance
```

Comment:
How tight is this bound? Right now we are seeing SSIM only failing sometime, because of randomness in input. I was wondering if we should set `atol=1e-5` just to be sure.

Reply:
I tested `1e-5` locally and it didn't pass, so increase to 6e-5

Reply:
Sorry I was confusing myself. Just tried running the test many times (with different seeds), seems to be stable now. Nicely done.